crates/sui-package-resolver/src/snapshots/sui_package_resolver__tests__pure_input_layouts_conflicting.snap

```diff
@@ -0,0 +1,19 @@
+---
+source: crates/sui-package-resolver/src/lib.rs
+expression: output
+---
+???
+u64
+???
+???
```

Comment:
Adding to your earlier question: This is the input that has conflicting usages (which caused the error previously). Now it shows up as having no discernable layout.